recipes/websphere.rb

```diff
@@ -0,0 +1,25 @@
+#
+# Cookbook Name:: websphere
+# recipe:: websphere
+#
+# Copyright (C) 2017 J Sainsburys
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+node.default['websphere_scripts_location'] = node['websphere']['location'] ? node['websphere']['location'] : '/tmp/websphere_scripts'
+
+directory node['websphere_scripts_location'] do
```

Comment:
We should add this to `websphere_base` or `websphere_cluster_members` instead, and change the default location from `/tmp` to `/usr/local/chef-websphere` or similar. `/tmp` may be tmpfs on many systems.